package.json

```diff
@@ -76,15 +75,12 @@
     "postcss-easy-import": "^3.0.0",
     "postcss-loader": "^4.2.0",
     "react": "^16.14.0",
```

Comment:
just one open question. Is it possible to upgrade the react version as in the other projects?